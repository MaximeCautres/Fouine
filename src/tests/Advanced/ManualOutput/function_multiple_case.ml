let f = function
  | x, y -> (prInt x; prInt y)
  | x -> prInt x
  | _ -> prInt 42
in
f (48, 5);
f 40
