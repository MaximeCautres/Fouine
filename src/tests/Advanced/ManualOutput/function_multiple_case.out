48
5
40
